### ISSUE TITLE
Do not hear ruler movement responses on mobile VO

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#233]
@zepumph, I thought all the ruler movement responses were implemented in #215, but it's possible I did not test mobile. 

While testing sound on mobile in issue #181 I noticed I was not hearing the region changes and hints to release on mobile. These alerts are outlined in this table in the design doc.

[Table: Pseudo Logic & Combined Phrases for Movement Responses (several examples)](https://docs.google.com/document/d/1-37qAgde2XrlXBQae2SgjartM35_EnzDD9pdtd3nXAM/edit#heading=h.2x1ofybk8azg)

I've got the missing responses highlighted in red.

@zepumph, was there a decision not to implement the region changes and release hints on mobile, or did they just fall through the cracks?

**Comment by terracoda** [comment]
Again, this issue is just about mobile. The responses are working great with keyboard on my MacBook Pro.

I tested this version on my iPhone 7 and didn't hear ruler movement responses:
https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.30/phet/gravity-force-lab_en_phet.html?a11y&supportsSound&supportsEnhancedSound

**Comment by zepumph** [comment]
Great catch @terracoda. I was calling the alert for the `KeyboardDragListener`, but not for the `DragListener` (for touch/mouse) which is what Gesture a11y relies on. Can you please test/review. Anything else here?

**Comment by terracoda** [comment]
Yes, I will test again. Please post a dev link. I need a dev link to test on my phone.

**Comment by terracoda** [comment]
@zepumph, sorry I forgot to tag you, I will need a dev link, but maybe the RC link will have this fix. I'll test that on my phone.

**Comment by terracoda** [comment]
I hear move responses and release hints on iOS now, yay! Closing.

**Comment by terracoda** [comment]
I tested the RC version - https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-rc.1/phet/gravity-force-lab_all_phet.html
